src/styles.css

```diff
@@ -260,14 +259,14 @@ body {
   'ins ins';
   grid-template-columns: 1fr 2fr;
   grid-template-rows: 1.5fr 2.5fr;
-  height: 80vh;
+  height: 86vh;
```

Comment:
Take a peek at that Codepen I sent you two the other day and see if that approach works. Using `vh/vw` can be a little unpredictable.

Reply:
Added the percentages in which worked perfectly thanks, the flex didn't work on our page flow given the number of elements we were using, but it worked without it, might have just done something wrong there. Definitely tucking that away for future reference, thanks again.